LICENSE

```diff
@@ -200,3 +200,9 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
```

Comment:
(Optional) I'm not sure what the official advice is, but I would do the following:

In the beginning of this doc, say:

> See docs/images/LICENSE for the license that applies to the files in the docs/images directory.
>
> The rest of `jj` is licensed under the terms below.

Then, I'd create the docs/images/LICENSE file with what you wrote here.

Reply:
I don't know what the best practice is either. I moved it to `docs/images/LICENSE`.

Reply:
If we get better advice, we can always adjust it.